lib/groupify/adapter/active_record/group.rb

```diff
@@ -149,10 +149,10 @@ def define_member_association(member_klass, association_name = nil)
 
           if ActiveSupport::VERSION::MAJOR > 3
```

Comment:
The conditional here can be removed since Rails 3 will no longer be supported or expected to work.